app/services/search_service.rb

```diff
@@ -183,4 +183,37 @@ def textSearch_questions(srchString)
     end
     sresult
   end
+
+  #Search nearby nodes with respect to given latitude and longitude
```

Comment:
I /think/ this needs a space between the `#` and `S` to be recognized by the auto API generator which I've been testing in another PR. Sorry to be so detail-oriented!

Reply:
No problem at all